src/test/java/api/support/http/ResourceClient.java

```diff
@@ -525,7 +525,7 @@ public void deleteAllIndividually()
 
     CompletableFuture<Response> getFinished = new CompletableFuture<>();
 
-    client.get(urlMaker.combine(""),
+    client.get(urlMaker.combine("?limit=1000"),
```

Comment:
Why has this limit been introduced?

Reply:
It was necessary to fetch all the records in tests (more that 10).

Reply:
Ok, I think we may want to change this to page through the records at some point.